### PR TITLE
add 1832-check-if-it-is-pangram

### PR DESCRIPTION
**Pull request by Rwatana**
Please check my code

### CODE REVIEW

File: 1832_check_if_it_is_a_pangram.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Solution:
+    def checkIfPangram(self, sentence: str) -> bool:
+        checker = [0] * 26
+        for i in sentence:
```
Comment (ryujumoto) [inline 1832_check_if_it_is_a_pangram.py]: "i" is used for index, so in this case it's better to use "character" or "letter".

**Review by ryujumoto — COMMENTED** [review]
Done.

**Comment by Rwatana** [comment]
I used the character instead of i. Please check my code.

File: 1832_check_if_it_is_a_pangram.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Solution:
```
Comment (ryujumoto) [inline 1832_check_if_it_is_a_pangram.py:5]: Write time complexity and space one.

**Comment by Rwatana** [comment]
I revised. Please check my code

**Review by ryujumoto — APPROVED** [review]

**Pull request merged**